.pre-commit-config.yaml

```diff
@@ -3,38 +3,31 @@ repos:
     rev: v1.2.3
     hooks:
       - id: trailing-whitespace
-        files: ^(packages|tests)/
         types: [file, python]
```

Comment:
I suggest that trailing-whitespace be removed in all files, not just python

Reply:
@andreyPromaster What about other `types: [file, python]`?

Reply:
I removed `types` from this hook